### PR TITLE
Hide forbidden actions in frontend, refs: #54811

### PR DESCRIPTION
**Pull request by lucashabersaat**

### CODE REVIEW

File: app/serializers/team_serializer.rb [hunk 1]
```diff
@@ -19,18 +19,22 @@
...
+  def deletable
+    TeamPolicy.new(user, object).destroy?
   end
 
   private
 
-  def user_id
-    current_user.id
+  def user
+    current_user
```
Comment (lucashabersaat) [inline app/serializers/team_serializer.rb:37]: such that i can mock the current user in the serializer tests

File: config/locales/ch_be.yml [hunk 2]
```diff
@@ -340,6 +340,7 @@ ch_be:
         user_not_logged_in: Benutzer isch nid igloggt
         invalid_request: Invalidi Ahfrag
         record_not_found: Ihtrag isch nid gfunde worde
+        delete_failed: Ihtrag het nid chenne g'löscht wärde
```
Comment (TheWalkingLeek) [inline config/locales/ch_be.yml:343]: Uiuiui das söu de aber bärndütsch si ;)

File: frontend/tests/unit/services/user-service-test.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | user-service', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let service = this.owner.lookup('service:user-service');
+    assert.ok(service);
+  });
```
Comment (TheWalkingLeek) [inline frontend/tests/unit/services/user-service-test.js:11]: Either put some real tests here or delete the whole file.
If the service doesn't have any logic, a unit test seems pretty useless :)

File: spec/system/teams_page_system_spec.rb [hunk 4]
```diff
@@ -7,6 +7,7 @@
 
 require 'rails_helper'
 
+require_relative '../../app/controllers/api/teams/members_controller'
```
Comment (TheWalkingLeek) [inline spec/system/teams_page_system_spec.rb:10]: Why is this needed?

File: spec/unit/serializers/team_serializer_spec.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+
+require 'rails_helper'
+
+describe TeamSerializer do
```
Comment (TheWalkingLeek) [inline spec/unit/serializers/team_serializer_spec.rb:10]: Niiiiiice!!!

File: spec/system/teams_page_system_spec.rb (hunk 4 above)
Comment (lucashabersaat) [reply spec/system/teams_page_system_spec.rb:10]: When testing this spec alone, the members_controller is not defined, however it works when doing all specs. Autoload is a bit messy and this line makes sure, it's loaded. Will insert a comment.
